psico/querying.py

```diff
@@ -488,6 +490,103 @@ def extinction_coefficient(selection="all", state=-1, *, quiet=1, _self=cmd):
     return (eps, A_280)
 
 
+@cmd.extendaa(cmd.auto_arg[1]['distance'], cmd.auto_arg[1]['distance'])
+def shortest_distance(selection1: str,
+                      selection2: str,
+                      state1: int = CURRENT_STATE,
+                      state2: int = CURRENT_STATE,
+                      name: str = "shortest",
+                      *,
+                      quiet: int = 1,
+                      _self=cmd):
+    '''
+DESCRIPTION
+    Finds the shortest pairwise distance between two selections.
+
+ARGUMENTS
+
+    sele_a = string: first atom selection
+
+    sele_b = string: second atom selection
+
+    sele_a_state = state of sele a {default: current state}
+
+    sele_b_state = state of sele b {default: current state}
+
+    obj_name = string: name of the object to create {default: shortest}
```

Comment:
can you fix the argument names in the docstring?

Reply:
Done. Sorry bout that.